inference_perf/__main__.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025
```

Comment:
```suggestion
Copyright 2025 The Kubernetes Authors.
```

same to all other headers